fedimint-api/src/db/mod.rs

```diff
@@ -252,6 +252,24 @@ impl<'a> DatabaseTransaction<'a> {
                 })
             })
     }
+
+    // TODO: this function doesn't seem very optimal
+    pub async fn remove_by_prefix<KP>(&mut self, key_prefix: &KP) -> Result<()>
```

Comment:
Would be cool if there was a default impl that can be overwritten if the DB supports this natively.

Reply:
All done.